app/models/search_result_child.rb

```diff
@@ -0,0 +1,14 @@
+SearchResultChild = Struct.new(:type, :id, :blurb) do
```

Comment:
A 'search result child' is a discussion, motion, or comment within a search result.

Each search result contains:

1 discussion
up to 2 proposals (the 2 most relevant to the search term)
up to 2 comments (the 2 most relevant to the search term)